lib/services/rdbmsManagement/README.md

```diff
@@ -0,0 +1,304 @@
+
+const subscriptionID = '<subscription id>';
+const resourceGroup = '<resource group name>';
+const serverName = '<server name>';
+
+msRestAzure.interactiveLogin().then((credentials) => {
+  let client = new MySQLManagementClient(credentials, subscriptionID);
+  return client.databases.listByServer(resourceGroup, serverName);
+}).then((databases) => {
+  console.log('List of databases:');
+  console.dir(databases, {depth: null, colors: true});
+}).catch((err) => {
+  console.log('An error ocurred');
+  console.dir(err, {depth: null, colors: true});
+});
+```
+
+#### Creating a MySQL database
+
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const MySQLManagementClient = require('azure-arm-rdbms').MySQLManagementClient;
+
+const subscriptionID = '<subscription id>';
+const resourceGroup = '<resource group name>';
+const serverName = '<server name>';
+const databaseName = '<database name>';
+
+msRestAzure.interactiveLogin().then((credentials) => {
+  let client = new MySQLManagementClient(credentials, subscriptionID);
+  return client.databases.createOrUpdate(resourceGroup, serverName, databaseName, {});
+}).then((database) => {
+  console.log('Database:');
+  console.dir(database, {depth: null, colors: true});
+}).catch((err) => {
+  console.log('An error ocurred');
+  console.dir(err, {depth: null, colors: true});
+});
+```
+
+#### Deleting a MySQL database
+
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const MySQLManagementClient = require('azure-arm-rdbms').MySQLManagementClient;
+
+const subscriptionID = '<subscription id>';
+const resourceGroup = '<resource group name>';
+const serverName = '<server name>';
+const databaseName = '<database name>';
+
+msRestAzure.interactiveLogin().then((credentials) => {
+  let client = new MySQLManagementClient(credentials, subscriptionID);
+  return client.databases.deleteMethod(resourceGroup, serverName, databaseName);
+}).then(() => {
+  console.log('Database deleted');
+}).catch((err) => {
+  console.log('An error ocurred');
+  console.dir(err, {depth: null, colors: true});
+});
+```
+
+#### Deleting a MySQL server
+
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const MySQLManagementClient = require('azure-arm-rdbms').MySQLManagementClient;
+
+const subscriptionID = '<subscription id>';
+const resourceGroup = '<resource group name>';
+const serverName = '<server name>';
+
+msRestAzure.interactiveLogin().then((credentials) => {
+  let client = new MySQLManagementClient(credentials, subscriptionID);
+  return client.servers.createOrUpdate(resourceGroup, serverName, {
+    location: 'eastus',
+    properties: {
+      createMode: 'Default',
+      administratorLogin: 'mysql',
+      administratorLoginPassword: 'F00Bar!!'
+    }
+  });
+}).then((server) => {
+  console.log('Server:');
+  console.dir(server, {depth: null, colors: true});
+}).catch((err) => {
+  console.log('An error ocurred');
+  console.dir(err, {depth: null, colors: true});
+});
+```
```

Comment:
Great set of examples on how to use the package. Even better with the usage of ES6 syntax, esp. with promises instead of callbacks :+1: 

My suggestion is, if possible, please add some typescript examples as well. Reasons:
1. This package is shipping with .d.ts files so typescript support is expected to be fully functional
2. Import syntax are better and different in typescript but there are also many ways to import. That depends on how your package exports things. So having examples would be great for this.
3. Since we are on Node 6.x, async/await is not fully supported yet. But Typescript fully supports async/await and compiles it down to whatever version of javascript the user prefers. This would make the promises chain code look even simpler.

It may not have to be as exhaustive as you have for `javascript` but having atleast a few would help :)